pointillism-info.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index 565c78a..69ea0e5 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -1,11 +1,18 @@
+ [Activity]
+ name = Pointillism
+-activity_version = 9
+-bundle_id = org.laptop.community.Pointillism
++activity_version = 9.1
++release_date = 2019-12-22
++metadata_license = CC0-1.0
++bundle_id = org.sugarlabs.Pointillism
+ exec = sugar-activity3 activity.Activity
+ show_launcher = yes
+ icon = activity
+ license = GPLv3+
+ summary = Create a pointillist image from the camera
++description = An incarnation of the pointillism activity live from your camera or a file.
+ repository = https://github.com/sugarlabs/pointillism.git
+-tags = Art
++tags = Graphics
++update_contact = srevin03@gmail.com
++developer_name = Sugar Labs Community
++url = https://github.com/sugarlabs/pointillism-activity#pointillism
++screenshots = https://user-images.githubusercontent.com/48695438/71541234-c19e4700-2966-11ea-8bbd-c42da59293c6.png
```

Comment:
Can you add more screenshots, including the original webcam mode? Please host it in a services that won't get deleted later.

Reply:
Unfortunately, there is no good pointillism screenshots using a camera except those on https://github.com/sugarlabs/pointillism-activity/pull/27#issuecomment-563453192 but I feel that, those images doesn't make any sense. If you get any such images, I will be glad to add them.